### ISSUE TITLE
json.Unmarshal error 

### ISSUE THREAD

**Issue by yuanshanhshan** [michimani/gotwi#255]
## Advise param from url
From twitter office url: https://developer.twitter.com/en/docs/twitter-api/tweets/timelines/api-reference/get-users-id-mentions#tab1
we can find return structure no have status.
![DeepinScreenshot_select-area_20220611162629](https://user-images.githubusercontent.com/84569701/173180256-9826da9d-c13d-4a8a-9f77-dd117e1a76ac.png)

when we request listMentions function like this：
`tt := &timelineType.ListMentionsInput{
		ID: twitterId,
		Expansions: fields.ExpansionList{
			fields.ExpansionAttachmentsMediaKeys,
			fields.ExpansionAuthorID,
		},
		TweetFields: fields.TweetFieldList{
			fields.TweetFieldEntities,    // the param can not to resolve
			fields.TweetFieldAuthorID,
		},
		MediaFields: fields.MediaFieldList{
			fields.MediaFieldUrl,
			fields.MediaFieldMediaKey,
		},
		UserFields: fields.UserFieldList{
			fields.UserFieldUsername,
		},
	}
`

**Comment by yuanshanhshan** [comment]
And this your URL :
`type URL struct {
	Start       *int    `json:"start"`
	End         *int    `json:"end"`
	URL         *string `json:"url"`
	ExpandedURL *string `json:"expanded_url"`
	DisplayURL  *string `json:"display_url"`
	Status      *string `json:"status"`
	Title       *string `json:"title"`
	Description *string `json:"description"`
	UnwoundURL  *string `json:"unwound_url"`
}`

**Comment by michimani** [comment]
@yuanshanhshan   
Thank you for your advice!

This was my mistake, the error was due to the `URL.Status` type being `*string` instead of `*int`. I will fix this.  
Please wait for a while until it is fixed.

Thank you again.

**Comment by michimani** [comment]
@yuanshanhshan   

Thank you for your patience. We have released v0.11.5 which resolves this issue.    

Also, in addition to fixing the part you pointed out, the `URL` structure was missing an `Images` field, so I added that as well. This seems to be missing from the official documentation as of today, so I have sent Twitter my feedback.

https://developer.twitter.com/en/docs/twitter-api/data-dictionary/object-model/tweet
